AUTHORING_GUIDE.md

```diff
@@ -171,6 +171,84 @@ comments and docstrings only as needed to further clarify the code’s intent.
 Always introduce functions and variables before they are used. Prefer less
 indirection. Prefer imperative programming as it is easier to understand.
 
+
+### Importing Google Cloud Libraries
+
+Follow this style for importing Google Cloud libraries:
+
+```py
+from google.cloud import texttospeech_v1
+```
+
+All commonly used clients and types are exposed under `texttospeech_v1`.
+
+```py
+from google.cloud import texttospeech_v1
+
+client = texttospeech_v1.TextToSpeechClient()
+
+audio_config = texttospeech.AudioConfig(
+        audio_encoding=texttospeech.AudioEncoding.MP3
+)
+```
+
+
+### Creating Request Objects for GAPICs
```

Comment:
I had trouble wrapping my head around where in the tree these sections belong.  Please let me know if you have suggestions on a better layout!

Reply:
I think the current layout is good for `Importing Google Cloud Libraries`. 

For `Creating Request Objects for GAPICs` section, maybe it makes more sense to put it between `Functions and Classes` and `README File` but since GAPIC instantiated objects aren't necessarily about functions/classes I think it's fine to put it where it is now.